doc/translation_process.md

```diff
@@ -44,11 +44,9 @@ Visit the [Transifex Signup](https://www.transifex.com/signup/) page to create a
 You can find the Bitcoin translation project at [https://www.transifex.com/bitcoin/bitcoin/](https://www.transifex.com/bitcoin/bitcoin/).
 
 ### Installing the Transifex client command-line tool
-The client is used to fetch updated translations. If you are having problems, or need more details, see [https://docs.transifex.com/client/installing-the-client](https://docs.transifex.com/client/installing-the-client)
+The client is used to fetch updated translations. If you are having problems, or need more details, see https://developers.transifex.com/docs/cli.
 
-`pip install transifex-client`
-
-Setup your Transifex client config as follows. Please *ignore the token field*.
+Setup your Transifex client config as follows. Please *ignore the token field* (unless you signed up associating your GitHub account, or another network account offered optionally by Transifex, which in that case the config file will be updated automatically with the provided token by the user once you execute the below mentioned script `update-translations.py` and a link to generate the token will be given).
```

Comment:
Nit (feel free to disregard) - this is a run-on sentence and complicated to follow. Please break it up to be more readable.

Reply:
True, I'll re-phrase it, thank you!

Reply:
I'll remove it entirely as it was also [pointed out](https://github.com/bitcoin/bitcoin/pull/29414#discussion_r1484148120) by @hebasto.